src/Symfony/Component/Validator/Constraints/All.php

```diff
@@ -17,7 +17,7 @@
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
  */
-#[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
+#[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::TARGET_PARAMETER | \Attribute::IS_REPEATABLE)]
```

Comment:
Updating all existing constraints to allow them to be put on parameters looks like a no-go to me. This will make it look like arbitrary method arguments could be validated while in fact requires some kind of manual validation. I would instead introduce some new attribute (which can only be added to method arguments) that will act as a container for validation constraints (similar to what classes implementing the `Composite` class already do).

Reply:
Yes, that's what I was afraid of :/
I'll look into the implementation of `Composite`.

Reply:
I created a new constraint called `ControllerArgument` and was inspired by the `Sequentially` constraint. It's way better now and at least I don't have to update most of the constraints. However, I'm not sure how to ensure that this constraint is only used on a controller argument. I guess its name should be enough?

Reply:
I can not say it is good idea. Why we can not add a `parameter` type on the existing Validator attributes?

If you have used Bean Validation in Java world(Spring WebMvc/WebFlux and JAX-RS/Jakarta REST), you will know put the attributes on method param directly is simple and clear.  

And the  Symfony Validator in concept is really a copy of Bean validation spec. https://beanvalidation.org/

**Introducing a new attribute to wrap the Validator attributes means increasing learning curve and inconsistent programming experience for the developers from other platforms.**